spaces/S000173/README.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: S000173
+slug: mathse-3145712
+name: Product of long ray and continuum power of a closed interval
+refs:
+  - mathse: 3145712
+    name: Answer to "Example of a locally compact + Hausdorff + ¬normal + connected space"
+---
+
+Tychonoff product of {S000038} and {S000103}. Used in
```

Comment:
Do we need the word "Tychonoff" here?

Reply:
Nah, I expect we simply use "product" elsewhere.